cedar-policy-core/src/ast/extension.rs

```diff
@@ -158,11 +158,11 @@ impl ExtensionFunction {
                 if args.is_empty() {
                     func()
                 } else {
-                    Err(evaluator::EvaluationError::WrongNumArguments {
-                        function_name: name.clone(),
-                        expected: 0,
-                        actual: args.len(),
-                    })
+                    Err(evaluator::EvaluationError::wrong_num_arguments(
+                        name.clone(),
+                        0,
+                        args.len(),
+                    ))
```

Comment:
side note that it's a shame this has to become less readable like this, I wish Rust had named arguments to functions